salsrc/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG OSPL_VERSION=6.9.0
+FROM lsstdm/salrun:${OSPL_VERSION} AS salidl-build
```

Comment:
The "AS" string seems incorrect and is unnecessary if you only have one stage.  Is this container still necessary at all?

Reply:
You are right, I noticed that after my last commit and already change it, but not pushed it. Thanks for noticing.